src/app/auth/login/page.tsx

```diff
@@ -0,0 +1,139 @@
+"use client";
+
+import { useRouter } from "next/navigation";
+import axios from "axios";
+import { useRef, useState } from "react";
+import Link from "next/link";
+import Button from '@/components/Button';
+import InputBox from "@/components/InputBox";
+import Image from "next/image"
+import loginValidation from "@/validations/LoginValidation";
+import OtpVerify from "@/components/otpVerify";
+
+export default function Login() {
+    const [email, setEmail] = useState('');
```

Comment:
Separate component logic from UI components. Use custom hooks as we have seen during capstone project.